confidence/confidence.py

```diff
@@ -151,6 +152,10 @@ async def resolve_object_details_async(
     ) -> FlagResolutionDetails[Union[Object, List[Primitive]]]:
         return await self._evaluate_async(flag_key, Object, default_value, self.context)
 
+    #
+    # --- internals
+    #
+
```

Comment:
Not familiar with Python enough maybe - does this mean logger setup can't be done my user?

Reply:
no, the `_` before method name is just a convention of internal methods which this is.